src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala

```diff
@@ -45,8 +45,15 @@ trait EntityPage extends HtmlPage {
   def headers: Elems = {
     def extScript(str: String) = Script(`type` = "text/javascript", src = str)
     def libScript(value: String) = extScript(relativeLinkTo(List(value, "lib")))
-
-    List(HtmlTags.Link(href = relativeLinkTo(List("index.css", "lib")), media = "screen", `type` = "text/css", rel = "stylesheet"),
+    val canonicalSetting = universe.settings.docCanonicalBaseUrl
+    val canonicalLink =  if (canonicalSetting.isSetByUser) {
+      val canonicalUrl =
+        if (canonicalSetting.value.endsWith("/")) canonicalSetting.value
+        else canonicalSetting.value + "/"
```

Comment:
is this normalization to end with a "/" always desired?

Reply:
The path to the actual page gets appended, so yes.

https://github.com/scala/scala/pull/7834/files#diff-789c4c9e5fea2476bc65c256a0449572R53